generator.go

```diff
@@ -66,20 +85,14 @@ func (t *Generator) CreateToken(data Data, options *Option) (string, error) {
 		return "", err
 	}
 
-	// generate the encoded headers
-	encodedHeader, err := encodedHeader()
-	if err != nil {
-		return "", err
-	}
-
 	// setup the claims for the token
 	claim := struct {
-		*Option
-		Version  int   `json:"v"`
-		Data     Data  `json:"d"`
-		IssuedAt int64 `json:"iat"`
+		Opts     *Option `json:"Options,omitempty"`
```

Comment:
This changes behavior.

This would result in the string that gets signed to be 
`{"Options":{"admin":true},"v":2,"d":{"hello":"world"},"iat":1257894000}`
Instead of 
`{"admin":true,"v":2,"d":{"hello":"world"},"iat":1257894000}`

http://play.golang.org/p/WYSPIoy2A3